### PR TITLE
Bump to 0.14

### PR DESCRIPTION
**Pull request by matzew**

### CODE REVIEW

File: deploy/resources/knative-eventing-kafka-contrib-v0.14.1.yaml [hunk 1]
```diff
@@ -423,6 +466,93 @@ data:
...
+        </buffer>
+      </match>
+    logging.revision-url-template: |
+      http://localhost:8001/api/v1/namespaces/knative-monitoring/services/kibana-logging/proxy/app/kibana#/discover?_a=(query:(match:(kubernetes.labels.knative-dev%2FrevisionUID:(query:'${REVISION_UID}',type:phrase))))
+    logging.request-log-template: '{"httpRequest": {"requestMethod": "{{.Request.Method}}", "requestUrl": "{{js .Request.RequestURI}}", "requestSize": "{{.Request.ContentLength}}", "status": {{.Response.Code}}, "responseSize": "{{.Response.Size}}", "userAgent": "{{js .Request.UserAgent}}", "remoteIp": "{{js .Request.RemoteAddr}}", "serverIp": "{{.Revision.PodIP}}", "referer": "{{js .Request.Referer}}", "latency": "{{.Response.Latency}}s", "protocol": "{{.Request.Proto}}"}, "traceId": "{{index .Request.Header "X-B3-Traceid"}}"}'
+    metrics.backend-destination: prometheus
+    metrics.request-metrics-backend-destination: prometheus
+    metrics.stackdriver-project-id: "<your stackdriver project id>"
+    metrics.allow-stackdriver-custom-metrics: "false"
+---
```
Comment (aliok) [inline deploy/resources/knative-eventing-kafka-contrib-v0.14.1.yaml:555]: Let's keep them. The operator discards these resources anyway

Comment (matzew) [reply deploy/resources/knative-eventing-kafka-contrib-v0.14.1.yaml:555]: yeap, it's a straight copy from upstream

Comment (aliok) [reply deploy/resources/knative-eventing-kafka-contrib-v0.14.1.yaml:555]: > Let's keep them. The operator discards these resources anyway

My bad. Operator doesn't discard these!

Comment (aliok) [reply deploy/resources/knative-eventing-kafka-contrib-v0.14.1.yaml:555]: https://github.com/openshift-knative/knative-kafka-operator/pull/57

**Pull request merged**